files/fingerprintjs2/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "fingerprintjs2",
+  "repo": "Valve/fingerprintjs2",
+  "files": {
+    "include": ["fingerprint2.js", "dist/fingerprint2.min.js"]
```

Comment:
I dont think that `fingerprint2.js` is required in production. Can you remove it?